test/clt-tests/core/test-fuzzy-search.rec

```diff
@@ -0,0 +1,569 @@
+| 10242 | ALEJANDRINA ZAPATA |
+| 10269 | JANNET QUINN       |
+| 10326 | JANICE APPEL       |
+| 10385 | JANN CASON         |
+| 10628 | LANETTE SWAIN      |
++-------+--------------------+
+––– input –––
+mysql -h0 -P9306 -e "SELECT * FROM name WHERE MATCH('EBGRNA') AND id BETWEEN 1000 AND 2000 ORDER BY username ASC OPTION fuzzy=1;"
+––– output –––
++------+---------------------+
+| id   | username            |
++------+---------------------+
+| 1353 | VERNA VITALE        |
+| 1783 | BERNADETTE HALES    |
+| 1838 | BERNARDINA MCMILLAN |
+| 1863 | ERNA ULLOA          |
+| 1947 | SAMIRA ABERNATHY    |
++------+---------------------+
+––– input –––
+mysql -h0 -P9306 -e "SELECT * FROM name WHERE MATCH('SARA') ORDER BY id ASC, username ASC OPTION fuzzy=1;"
+––– output –––
++------+------------------+
+| id   | username         |
++------+------------------+
+|  187 | LARA KIRK        |
+|  206 | CLARA MORRIS     |
+|  350 | KLARA DONALDSON  |
+|  989 | ELAINE SARABIA   |
+| 1408 | LARA FREDERICK   |
+| 1660 | SARA GARNER      |
+| 1819 | KARA CINTRON     |
+| 1830 | SARAI ESPINAL    |
+| 2073 | VALENTIN SARABIA |
+| 2929 | LARA CINTRON     |
+| 3044 | KLARA ANDRES     |
+| 3372 | KLARA NORTON     |
+| 3740 | CRUZ SARABIA     |
+| 3779 | RUSSEL LARA      |
+| 4699 | HEE SARABIA      |
+| 4932 | KLARA GOODING    |
+| 5829 | SARAI LINDLEY    |
+| 5864 | SARAN KNAPP      |
+| 5917 | SARA WAITE       |
+| 6634 | CLARA BEALE      |
++------+------------------+
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 1 AS fuzziness);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 'us,ru' AS layouts);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 1 AS append);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 1 AS prepend);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 1 AS prepend, 1 AS append);" | tail -n +4 | sort
+––– output –––
++---------+
+| joe     |
+| john    |
+| johnson |
+| jonas   |
+| jordan  |
+| jose    |
+| joseph  |
+| joy     |
+| joyce   |
+| major   |
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('jo', 'name', 1 AS fuzziness, 'us,ru' AS layouts, 1 AS append, 1 AS prepend);" | tail -n +4 | sort
```

Comment:
We can pass an empty string ('') as layouts because when we don't need to test it, there's no reason to always use it.

It's sufficient to use 2-3 extra queries with different layouts to ensure nothing has changed, but there's no point in adding them to every type of test.